src/app/containers/ArticleTimestamp/index.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
```

Comment:
Great work on the refactor. This Article Timestamp container is much more readable. ✨